build.gradle

```diff
@@ -22,7 +22,7 @@ if (ciBuild) {
 
 jenkinsPlugin {
   // Version of Jenkins core this plugin depends on.
-  coreVersion = '1.642.1'
+  coreVersion = '1.651.3'
```

Comment:
Is the core update required?

Reply:
It was to install the `Git` plugin which I used for local testing. It's the current 1.6xx LTS so seemed more appropriate anyhow vs 1.642.1 which is a bit of an arbitrary version. I don't think from an API standpoint I need anything from that newer version though. We could back this out.